javascript/javascript2/week1/spiritAnimalGenerator/index.js

```diff
@@ -0,0 +1,40 @@
+const spiritAnimalNames = ['The humming bee', 'The cheeky monkey', 'The crazy frog', 'The beautiful butterfly', 'The fullmoon wolf', 'The angry lion', 'The poisonous snake', 'The sqeaky hedgehog', 'The elegant swan', 'The greedy fox'];
+
+
+const myButton = document.querySelector('button');
+const spiritName = document.querySelector('p');
+
+
+function clickButton(){
+const inputName = document.getElementById('name').value;
+if(inputName !== ''){
+    spiritName.innerHTML = getSpiritName(inputName);
+}else{
+    spiritName.innerHTML = 'Please provide a name';
+}
```

Comment:
`spiritName.innerHTML = inputName ?  getSpiritName(inputName) : 'Please provide a name';`

Reply:
I changed it to ternary operator